src/receivers/custom-routes.ts

```diff
@@ -0,0 +1,48 @@
+import { IncomingMessage, ServerResponse } from 'http';
```

Comment:
Separated the route-specific logic into a new file, per @seratch's suggestion, as well as cleaned up the logic a bit.